src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1207,32 +1208,119 @@ void missingRdfTypeOnNamedNode() {
         }
     }
 
+    class BaseExample {
+        Model model;
+        Resource system;
+        ValidateAIF validator = seedlingValidator;
+
+        void setup() {
+            model = createModel();
+            system = makeSystemWithURI(model, getTestSystemUri());
+        }
+
+        void addType(Resource resource, Resource type) {
+            markType(model, getAssertionUri(), resource, type, system, 1d);
+        }
+
+        void testInvalid(String name) {
+            assertAndDump(model, name, validator, false);
+        }
+
+        void testValid(String name) {
+            assertAndDump(model, name, validator, true);
+        }
+
+    }
+
     /**
      * Set of tests to show that NIST restrictions pass and fail appropriately
      */
     @Nested
-    class NISTExamples {
-        Model model;
-        Resource system;
+    class NISTExamples extends BaseExample {
         Resource entity;
         Resource event;
+        Resource relation;
         Resource entityCluster;
         Resource eventCluster;
+        Resource relationCluster;
+        NISTExamples() { validator = nistSeedlingValidator; }
 
-        void addType(Resource resource, Resource type) {
-            markType(model, getAssertionUri(), resource, type, system, 1d);
-        }
 
         @BeforeEach
         void setup() {
-            model = createModel();
-            system = AIFUtils.makeSystemWithURI(model, getTestSystemUri());
+            super.setup();
+
             entity = AIFUtils.makeEntity(model, getEntityUri(), system);
             addType(entity, SeedlingOntology.Person);
             event = AIFUtils.makeEvent(model, getUri("event1"), system);
             addType(event, SeedlingOntology.Conflict_Attack);
+            relation = makeRelation(model, getUri("relation1"), system);
+            addType(relation, SeedlingOntology.GeneralAffiliation_APORA);
             entityCluster = makeClusterWithPrototype(model, getClusterUri(), entity, system);
             eventCluster = makeClusterWithPrototype(model, getClusterUri(), event, system);
+            relationCluster = makeClusterWithPrototype(model, getClusterUri(), relation, system);
+        }
+
+        // Each edge justification must be represented uniformly in AIF by
+        // aida:CompoundJustification, even if only one span is provided
+        // Edges are assumed to be relation and event arguments
+        //
+        // CompoundJustification must be used only for justifications of argument assertions,
+        // and not for justifications for entities, events, or relation KEs
+        @Nested
+        class RestrictJustification {
+            @Test
```

Comment:
It looks like you moved `RestrictJustification` to the top of the class definition; it used to be at the bottom after `ConfidenceValueRange`.  Was this deliberate, and if so, why?  It just causes a bunch of additional diffs, but if there's a reason, then no worries.

Also, I'm not really reviewing the content of `RestrictJustification` and `ClusterHasIRI` although in theory I should, since we're re-adding it to the tree.

Reply:
I wanted to keep similar tests together. This is now at the top to be near EdgeJustificationLimit. If you feel that is too confusing, I'm OK to move it back

Reply:
No not at all-- I just wanted to make sure there was some kind of reason and it wasn't some kind of side effect of merge-error-fixing.  Works for me.